src/devices/acurite.c

```diff
@@ -216,8 +216,23 @@ static int acurite_rain_gauge_callback(bitbuffer_t *bitbuffer) {
     if ((bb[0][0] != 0) && (bb[0][1] != 0) && (bb[0][2]!=0) && (bb[0][3] == 0) && (bb[0][4] == 0)) {
 	    float total_rain = ((bb[0][1]&0xf)<<8)+ bb[0][2];
 		total_rain /= 2; // Sensor reports number of bucket tips.  Each bucket tip is .5mm
+/*
         fprintf(stdout, "AcuRite Rain Gauge Total Rain is %2.1fmm\n", total_rain);
 		fprintf(stdout, "Raw Message: %02x %02x %02x %02x %02x\n",bb[0][0],bb[0][1],bb[0][2],bb[0][3],bb[0][4]);
+*/
+		uint8_t id = bb[0][0];
```

Comment:
please fix indentation.

Reply:
what is prefered indentation? I use tabsize=4, but I also see spaces in file so I'm confused.